<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="P3B-6LHCNK2" name="Test 1" tests="21" failures="5" timestamp="10 Dec 2019 11:06:56 GMT" time="22.721" errors="0">
  <testcase name="test1" time="0.006" classname="testNG.demoassertion"/>
  <testcase name="test2" time="0.002" classname="testNG.demoassertion">
    <failure type="java.lang.AssertionError" message="Error msg- 20 is greater than 18 expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Error msg- 20 is greater than 18 expected [false] but found [true]
	at testNG.demoassertion.test2(demoassertion.java:17)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase name="test3" time="0.001" classname="testNG.demoassertion"/>
  <testcase name="test4" time="0.002" classname="testNG.demoassertion">
    <failure type="java.lang.AssertionError" message="expected [Hi] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hi] but found [Hello]
	at testNG.demoassertion.test4(demoassertion.java:26)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- test4 -->
  <testcase name="test5" time="0.002" classname="testNG.demoassertion"/>
  <testcase name="test6" time="0.003" classname="testNG.demoassertion"/>
  <testcase name="test7" time="0.002" classname="testNG.demoassertion">
    <failure type="java.lang.AssertionError" message="expected object to not be null">
      <![CDATA[java.lang.AssertionError: expected object to not be null
	at testNG.demoassertion.test7(demoassertion.java:40)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- test7 -->
  <testcase name="test8" time="0.003" classname="testNG.demoassertion">
    <failure type="java.lang.AssertionError" message="Intentionally failing the condition">
      <![CDATA[java.lang.AssertionError: Intentionally failing the condition
	at testNG.demoassertion.test8(demoassertion.java:44)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- test8 -->
  <testcase name="f" time="0.001" classname="testNG.demodataprovider"/>
  <testcase name="f" time="0.001" classname="testNG.demodataprovider"/>
  <testcase name="f" time="0.001" classname="testNG.demodataprovider"/>
  <testcase name="test1" time="0.007" classname="testNG.demosoftassert">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Hi] but found [Hello],
	Error message:  both are equal">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [Hi] but found [Hello],
	Error message:  both are equal
	at testNG.demosoftassert.test1(demosoftassert.java:20)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase name="f" time="0.0" classname="testNG.executionsequence"/>
  <testcase name="f" time="6.632" classname="testNG.LoginMercury"/>
  <testcase name="f" time="0.003" classname="testNG.NewTest"/>
  <testcase name="openBrowser" time="0.001" classname="testNG.Mercuryapplication"/>
  <testcase name="openURL" time="0.001" classname="testNG.Mercuryapplication"/>
  <testcase name="Login" time="0.001" classname="testNG.Mercuryapplication"/>
  <testcase name="searchFlight" time="0.001" classname="testNG.Mercuryapplication"/>
  <testcase name="bookFlight" time="0.002" classname="testNG.Mercuryapplication"/>
  <testcase name="closeBrowser" time="0.001" classname="testNG.Mercuryapplication"/>
</testsuite> <!-- Test 1 -->
